xbmc/interfaces/json-rpc/schema/types.json

```diff
@@ -109,7 +109,7 @@
                   "album", "albumtype", "genre", "country", "year", "rating", "userrating", "votes", "top250", "programcount",
                   "playlist", "episode", "season", "totalepisodes", "watchedepisodes", "tvshowstatus", "tvshowtitle",
                   "sorttitle", "productioncode", "mpaa", "studio", "dateadded", "lastplayed", "playcount", "listeners",
-                  "bitrate", "random", "totaldiscs", "originaldate", "bpm" ]
+                  "bitrate", "random", "totaldiscs", "originaldate", "bpm", "originaltitle" ]
```

Comment:
You also need to bump the minor part of the JSON API version.

And please fix the commit message. It should at least mention `JSON-RPC` and `sort by original title`.

Reply:
Better now?

Reply:
Yes as long as you squash the three commits concerning JSON-RPC into one :+1:

Reply:
Done :)